### PR TITLE
fix(mobileSettingSwipe): close setting modal upon refresh

### PR DESCRIPTION
**Pull request by KemoPaw**
**What this PR does** 📖

Closes mobile Settings Modal upon user entering the chat in mobile. 

**Which issue(s) this PR fixes** 🔨

[AP-936](https://satellite-im.atlassian.net/jira/software/c/projects/AP/boards/5?modal=detail&selectedIssue=AP-936)

**Special notes for reviewers** 🗒️
The issue of what was causing the bug before was that the mobile-chat's swipeHandler would attach itself to the mobile-setting's modal. I'm not too sure if this is the best way to handle this bug but am open to suggestions/improvements!

**Additional comments** 🎤

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -128,7 +128,7 @@
     "dotenv-cli": "^4.0.0",
     "electron": "^13.2.1",
     "encoding": "^0.1.13",
-    "eslint": "^8.3.0",
+    "eslint": "^8.10.0",
```
Comment (stavares843) [inline package.json:131]: any specific reason why was this updated? 🔨

Comment (KemoPaw) [reply package.json:131]: @stavares843 Was just trying to run eslint and it decided to update. I can remove it if you'd like though!

Comment (Jekrimo) [reply package.json:131]: Yeah, will you put it to 8.3.0 again? Just encase honestly, but if there is something going on locally for you that isn't working that an update would fix let us know

**Review by Jekrimo — APPROVED** [review]

**Review by phillsatellite — APPROVED** [review]

**Pull request merged**